src_modules/player/commands/ShuffleCommand.ts

```diff
@@ -0,0 +1,20 @@
+import { Command, CommandContext } from '../../../commands/command';
+import { PlayerModule } from '../index';
+
+export class ShuffleCommand extends Command {
+  constructor() {
+    super({
+      name: 'shuffle',
+      description: 'Shuffle the player queue',
+      module: 'player',
+    });
+  }
+
+  async execute(ctx: CommandContext) {
+    const player = await PlayerModule.get(ctx.bot, ctx.msg);
```

Comment:
You cannot get the playermodule like this. Instead add a new property to the command class, and add a parameter to the constructor. Similar to ClearCommand.ts.